Toolkit/ArcGISToolkit/TimeSlider.swift

```diff
@@ -542,11 +542,24 @@ public class TimeSlider: UIControl {
     // MARK: Other Properties
     
     /**
-     This will initialize the slider's fullExtent, currentExtent and timeStepInterval
-     properties based on time enabled layers in the geo view.
+     The geo view used to initialize the time slider properties.
```

Comment:
"geo view" is usually one word in the doc:  `geoView`.

Reply:
fixed